falafel/settings.py

```diff
@@ -1,32 +1,43 @@
 import sys
 import os
 import yaml
+import pkgutil
 
 INSTALL_DIR = os.path.dirname(os.path.abspath(__file__))
 NAME = "falafel.yaml"
 DEFAULTS_NAME = "defaults.yaml"
 
 PATHS = [
-    os.path.join(INSTALL_DIR, DEFAULTS_NAME),  # Defaults
-    os.path.join("/etc", NAME),  # System-wide config
-    os.path.join(os.path.expanduser("~/.local"), NAME),  # User-specific config
-    "." + NAME  # Directory-specific config
+    ("resource", DEFAULTS_NAME),  # Defaults
```

Comment:
@songuyen I just wanted to confirm that you no longer need to join `INSTALL_DIR` to `DEFAULTS_NAME`?